README.md

```diff
@@ -37,6 +37,11 @@ You can change user or roles by setting environment variables:
 REMOTE_USER=auser@stanford.edu ROLES=dlss:hydrus-app-administrators bin/dev
 ```
 
+Note: if you need to force a deposit completing in localhost development (to get around workflows not running), run this rake task, which will just set all
+pending deposits (items and collections) to completed.  Imporant: only for localhost...
+
+`rake complete_deposits`
+
```

Comment:
FWIW, this is documented on the app wiki: https://github.com/sul-dlss/happy-heron/wiki/Complete-deposits-locally

The README may be a better location, though. Worth taking the wiki stuff and merging it in here?

Reply:
Cool.  How about a single line in the README that just points to the wiki page (to avoid duplication but at least help you find it if you just look at the README)

Reply:
@peetucket works for me!

Reply:
switched text to link to wiki